### ISSUE TITLE
electrum.interface.RequestCorrupted: fees must be in decreasing order

### ISSUE THREAD

**Issue by emanuelelaface** [romanz/electrs#317]
I just updated Electrum to the latest master (4.0.4-54-g1c07777e1) and when I run it on latest Electrs server (v0.8.5-32-g8f2f533) this is what I have
```
E/i | interface.[electrum.scumm.it:50003] | Exception in wrapper_func: RequestCorrupted('fees must be in decreasing order')
Traceback (most recent call last):
  File "/Users/ema/Dropbox/python-projects/electrum/electrum/util.py", line 1039, in wrapper
    return await func(*args, **kwargs)
  File "/Users/ema/Dropbox/python-projects/electrum/electrum/interface.py", line 480, in wrapper_func
    return await func(self, *args, **kwargs)
  File "/Users/ema/Dropbox/python-projects/electrum/electrum/interface.py", line 504, in run
    await self.open_session(ssl_context)
  File "/Users/ema/Dropbox/python-projects/electrum/electrum/interface.py", line 647, in open_session
    await group.spawn(self.monitor_connection)
  File "/Users/ema/anaconda3/lib/python3.7/site-packages/aiorpcX-0.18.3-py3.7.egg/aiorpcx/curio.py", line 242, in __aexit__
    await self.join()
  File "/Users/ema/anaconda3/lib/python3.7/site-packages/aiorpcX-0.18.3-py3.7.egg/aiorpcx/curio.py", line 211, in join
    raise task.exception()
  File "/Users/ema/Dropbox/python-projects/electrum/electrum/network.py", line 452, in _request_server_info
    await group.spawn(self._request_fee_estimates(interface))
  File "/Users/ema/anaconda3/lib/python3.7/site-packages/aiorpcX-0.18.3-py3.7.egg/aiorpcx/curio.py", line 242, in __aexit__
    await self.join()
  File "/Users/ema/anaconda3/lib/python3.7/site-packages/aiorpcX-0.18.3-py3.7.egg/aiorpcx/curio.py", line 211, in join
    raise task.exception()
  File "/Users/ema/Dropbox/python-projects/electrum/electrum/network.py", line 456, in _request_fee_estimates
    histogram = await interface.get_fee_histogram()
  File "/Users/ema/Dropbox/python-projects/electrum/electrum/interface.py", line 1004, in get_fee_histogram
    raise RequestCorrupted(f'fees must be in decreasing order')
electrum.interface.RequestCorrupted: fees must be in decreasing order
```
The client disconnects and connects continuously with this error.

**Comment by romanz** [comment]
Thanks for reporting this issue!

**Comment by romanz** [comment]
Should be fixed by c88a0dc
